packages/ui-kit/src/components/Counter/Counter.stories.tsx

```diff
@@ -28,25 +34,27 @@ const connectedParams = {
   }
 }
 
-const ConnectedCounterTemplate = (args: Story['args']) => {
+const Counter = (args: Story['args']) => {
   const { accessToken } = useStorybookAccessToken(
```

Comment:
I think that is previous PR scope, But it seems not necessary passing param for 'useStorybookAccessToken' customHook. Because it is passed every time same constant values. This env params can be readen in the customHook.